### ISSUE TITLE
[SUGGESTION] Add Deep Resonance

### ISSUE THREAD

**Issue by SireTeddy** [itsdinkd/AQM3#51]
Hello,
I'm here because i'm a fan of this old mod : Deep Resonance, and it was abandonned since a few years, but since few weeks, it's now updated again ! 
it's a good mod for begginners and in middle game (often for create your own crystals).
i think you can look the mods with this link : https://www.curseforge.com/minecraft/mc-mods/deep-resonance

**Comment by itsdinkd** [comment]
i am still debating this. talk me into it please

**Comment by SireTeddy** [comment]
Yeah i'm ready to talk with you for that. The mod is updated recently, but just for new versions, not new functions, and the mod has the old textures of ore etc...
